### PR TITLE
🐛 Slot index and token index must start at 0

### LINKED ISSUE #31: Slot index must start at 0

### PR DESCRIPTION
**Pull request by bal7hazar**
Fix #31

### CODE REVIEW

File: protostar.toml [hunk 1]
```diff
@@ -12,3 +12,6 @@ ERC3525Full = ["./src/erc3525/presets/ERC3525Full.cairo"]
 ERC3525MintableBurnable = ["./src/erc3525/presets/ERC3525MintableBurnable.cairo"]
 ERC3525SlotApprovable = ["./src/erc3525/presets/ERC3525SlotApprovable.cairo"]
 ERC3525SlotEnumerableMintableBurnable = ["./src/erc3525/presets/ERC3525SlotEnumerableMintableBurnable.cairo"]
+
+[test]
+target = ["tests"]
```
Comment (tekkac) [inline protostar.toml:17]: +1

File: tests/unit/test_slot_enumerations.cairo [hunk 2]
```diff
@@ -172,22 +172,14 @@ func test_token_index_reverts_if_index_invalid{
     return ();
 }
 
-@view
-func test_token_index_zero_nonexistent{
```
Comment (tekkac) [inline tests/unit/test_slot_enumerations.cairo:176]: This should now test that SafeMath does its job when using the index `Uint256(2**128-1, 2**128-1)` 
(-1 as Uint256)

**Review by tekkac — DISMISSED** [review]
Seems to work great, OZ fanbois win.
Minor comment on unit testing.
:muscle:

File: tests/unit/test_slot_enumerations.cairo (hunk 2 above)
Comment (bal7hazar) [reply tests/unit/test_slot_enumerations.cairo:176]: Done for token in slot by index and slot by index

**Review by tekkac — APPROVED** [review]

**Pull request merged**